src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -175,15 +184,36 @@ def load_arguments(self, _):
         c.argument('container_name', container_name_type, id_part='child_name_2')
         c.argument('item_name', item_name_type, id_part='child_name_3')
 
-    for command in ['list', 'show-log-chain']:
-        with self.argument_context('backup recoverypoint ' + command) as c:
-            c.argument('vault_name', vault_name_type, id_part=None)
-            c.argument('start_date', type=datetime_type, help='The start date of the range in UTC (d-m-Y).')
-            c.argument('end_date', type=datetime_type, help='The end date of the range in UTC (d-m-Y).')
-            c.argument('backup_management_type', backup_management_type)
-            c.argument('container_name', container_name_type)
-            c.argument('workload_type', workload_type)
-            c.argument('use_secondary_region', action='store_true', help='Use this flag to list recoverypoints in secondary region.')
+    with self.argument_context('backup recoverypoint list') as c:
+        c.argument('vault_name', vault_name_type, id_part=None)
+        c.argument('start_date', type=datetime_type, help='The start date of the range in UTC (d-m-Y).')
+        c.argument('end_date', type=datetime_type, help='The end date of the range in UTC (d-m-Y).')
+        c.argument('backup_management_type', backup_management_type)
+        c.argument('container_name', container_name_type)
+        c.argument('workload_type', workload_type)
+        c.argument('use_secondary_region', action='store_true', help='Use this flag to list recoverypoints in secondary region.')
+        c.argument('is_ready_for_move', arg_type=get_three_state_flag(), help='Use this flag to retrieve the recoverypoints that are ready to be moved to destination-tier.')
+        c.argument('target_tier', target_tier_type)
+        c.argument('tier', tier_type)
+        c.argument('recommended_for_archive', action="store_true", help='Use this flag to retrieve recommended archivable recoverypoints.')
+
+    with self.argument_context('backup recoverypoint move') as c:
+        c.argument('vault_name', vault_name_type, id_part=None)
+        c.argument('container_name', container_name_type)
+        c.argument('rp_name', rp_name_type, id_part='child_name_4')
```

Comment:
rename the param to 'name'. I know we thought --rp-id but I'm now thinking it should be same in both 'az backup recoverypoint show/move'. Check with Kartik too

Reply:
done